frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -347,6 +333,21 @@ const HostDetailsPage = ({
     }
   );
 
+  const canEditMdm = (() => {
+    const userHasPermission =
+      !!currentUser &&
+      !!host &&
+      (isGlobalAdmin ||
+        isGlobalMaintainer ||
+        permissionUtils.isTeamMaintainerOrTeamAdmin(currentUser, host.team_id));
+    const hostEnrolled = ["On (automatic)", "On (manual)"].includes(
+      host?.mdm_enrollment_status ?? ""
+      // TODO: API will be reverted to nested mdm {enrollment_status: ...}, change to below when it is:
+      // host.mdm?.enrollment_status ?? ""
```

Comment:
Looks like this will be done today if you want to hold: https://github.com/fleetdm/fleet/pull/9505